nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/spi/DnsResolverProvider.java

```diff
@@ -14,12 +14,25 @@
  * limitations under the License.
  */
 
-package io.helidon.nima.http2.webclient;
+package io.helidon.nima.webclient.spi;
 
-import io.helidon.nima.common.tls.Tls;
+/**
+ * Provider interface for custom DNS resolvers.
+ */
+public interface DnsResolverProvider {
```

Comment:
This is an SPI and it is called `Provider`, yet I do not see any `uses` statement in `module-info.java`. 
If this is an omission, please fix.
If this is by design (and users are expected to provide an instance), we need to use factory methods for implementations and not default constructors.